samples/highcharts/demo/venn-point-legend/demo.js

```diff
@@ -0,0 +1,53 @@
+Highcharts.chart('container', {
+    accessibility: {
+        point: {
+            descriptionFormatter: function (point) {
+                var intersection = point.sets.join(', '),
+                    name = point.name,
+                    ix = point.index + 1,
+                    val = point.value;
+                return ix + '. Intersection: ' + intersection + '. ' +
+                    (point.sets.length > 1 ? name + '. ' : '') + 'Value ' + val + '.';
+            }
+        }
+    },
+    series: [{
+        showInLegend: true,
+        legendType: 'point',
```

Comment:
Since this is the default `legendType` for venn (as of this PR), it should not be part of the config. It is also not documented in the API since it has the `@ignore-option` tag.